grafana-plugin/playwright.config.ts

```diff
@@ -32,16 +32,16 @@ const config: PlaywrightTestConfig = {
     timeout: 10000,
   },
   /* Run tests in files in parallel */
-  fullyParallel: true,
+  fullyParallel: false,
   /* Fail the build on CI if you accidentally left test.only in the source code. */
-  forbidOnly: !!process.env.CI,
+  forbidOnly: IS_CI,
   /**
    * Retry on CI only
    *
    * NOTE: until we fix this issue (https://github.com/grafana/oncall/issues/1692) which occasionally leads
-   * to flaky tests.. let's just retry failed tests. If the same test fails 3 times, you know something must be up
+   * to flaky tests.. let's allow 1 retry per test
    */
-  retries: !!process.env.CI ? 3 : 0,
+  retries: IS_CI ? 1 : 0,
```

Comment:
I had a lot of failing runs in past days and I didn't notice that if the first retry fails, any other retry would pass. And more retries makes the pipeline longer